### ISSUE TITLE
inetutils ping has incorrect permission

### ISSUE THREAD

**Issue by ghost** [LnL7/nix-darwin#180]
Hi,

I listed `inetutils` in my `environment.systemPackages` and `darwin-rebuild switch` worked without problem. However, the `ping` utility is not working properly:
```
ping: Lacking privilege for raw socket.
```

It seems that `nix-darwin` doesn't support security wrapper. Is there any workaround for this problem?

**Comment by thanegill** [comment]
I ran into the same issue. Security wrappers seem to be the correct solution. But for now this has worked for me:

```nix
system.activationScripts.applications.text = ''
    chmod u+s ${pkgs.inetutils}/bin/ping
  '';
```

**Comment by Enzime** [comment]
Closing in favour of #618

**Comment by Atry** [comment]
> I ran into the same issue. Security wrappers seem to be the correct solution. But for now this has worked for me:
> 
> ```nix
> system.activationScripts.applications.text = ''
>     chmod u+s ${pkgs.inetutils}/bin/ping
>   '';
> ```

Can we make a module similar to https://github.com/NixOS/nixpkgs/blob/7eee17a8a5868ecf596bbb8c8beb527253ea8f4d/nixos/modules/security/wrappers/default.nix

**Comment by Enzime** [comment]
There's a PR available at #890
